README.md

```diff
@@ -174,7 +174,7 @@ I would like to thank Microverse Family and all my coding partners.
 
 MIT License
 
-Copyright (c) [year] [fullname]
+Copyright (c) 2023 Joyce Cruz and Mehmet Selçuk Güler
 
 Permission is hereby granted, free of charge, to any person obtaining a copy
 of this software and associated documentation files (the "Software"), to deal
```

Comment:
- [ ] Please take note that the "License" section should include a link to the License file that you've created in the root of your project and shouldn't include the license documentation itself so please remove it and instead add the link to the file as shown in the [Microverse template](https://github.com/microverseinc/readme-template/blob/master/README.md)
- Here's how it should look:
  ![image](https://github.com/s2jack/awesome-books/assets/104968236/c5c70860-a8bc-4b61-9d65-1e7fe0d92a70)